src/components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import './button.css'
+
+const Button = ({ name, styleForButton }) => {
+    return (
+        <div>
+            <button className={styleForButton}>
+                {name}
```

Comment:
Instead of passing in `name` you should use `children`, so you can use it like a normal button,
```
<Button> Moj button </Button>
```